apps/docs/app/tokens/components/Aside/aside.css

```diff
@@ -0,0 +1,53 @@
+.hd-aside {
+    display: flex;
+    flex-direction: column;
+    padding-block: var(--hd-space-8);
+    user-select: none;
+    width: 10rem;
+    position: sticky;
+    top: var(--hd-space-8);
+    height: calc(100% - var(--hd-space-1));
+}
+
+.hd-aside__list {
+    list-style-type: none;
+    padding: 0;
+    display: flex;
+    flex-direction: column;
+    gap: var(--hd-space-05);
+}
+
+.hd-aside-section__title {
+    display: flex;
+    font-size: 0.875rem;
+    font-weight: 600;
+    margin-block-end: var(--hd-space-1);
+    color: var(--hd-color-neutral-text);
+}
+
+.hd-aside__list .hd-aside__list {
+    border-inline-start: 1px solid var(--hd-color-neutral-border);
+}
+
+.hd-aside__list-item {
+    padding-block: var(--hd-space-05);
+    font-size: 0.875rem;
+    position: relative;
+}
+
+.hd-aside__list .hd-aside__list .hd-aside__list-item--active::before {
```

Comment:
As far as possible, we don't have more than two children. Can you just use `.hd-aside__list-item--active::before`

Reply:
I need to make sure we are two level deep in the ul li ul li